bundles.json

```diff
@@ -18,13 +18,13 @@
     "platform/features/scrolling",
     "platform/features/events",
     "platform/forms",
+    "platform/persistence/elastic",
     "platform/persistence/queue",
     "platform/policy",
     "platform/entanglement",
     "platform/search",
 
     "example/imagery",
-    "example/persistence",
     "example/eventGenerator",
```

Comment:
@slhale Can we revert this change before merge (e.g. stick with example/persistence in the default configuration?)

Alternately, I'd be okay with `example/localstorage` at this point (and think that is reasonably relevant here since it provides an initial set of domain objects to index for search...)